napari/utils/misc.py

```diff
@@ -441,3 +449,55 @@ def dir_hash(path: Union[str, Path], include_paths=True, ignore_hidden=True):
                 _hash.update(''.join(fparts).encode())
 
     return _hash.hexdigest()
+
+
+def _generate_cls_stubs(cls, output_file, imports: Sequence[str] = ()) -> str:
+    """Generate pyi-format type stubs for a class."""
+    import textwrap
+
+    from ..utils.misc import get_subclass_methods
```

Comment:
I'm a bit confused here, aren't both of these functions in `napari.utils.misc`?

Reply:
hah.  they weren't at first :)  thanks for catching!